server/xpub-model/entities/user/helpers/elife-api.js

```diff
@@ -46,16 +46,19 @@ const convertPerson = apiPerson => {
 
 const people = async role => {
   logger.debug('Fetching editors from /people', { role })
+
   let items = []
   let response
   let page = 1
   do {
-    response = await request('people', {
-      order: 'asc',
-      page,
-      'per-page': 100,
-      type: role,
-    })
+    let query = `order=asc&page=${page}&per-page=100`
```

Comment:
If anyone wants to suggest a library for doing this feel free... but its pretty simple to do.

Reply:
We can use `qs` to convert the query object to a string before passing it along to `superagent`. [It handles arrays as well](https://github.com/visionmedia/superagent/issues/629#issuecomment-109197779)

Reply:
See conversation: https://github.com/elifesciences/elife-xpub/pull/1301#discussion_r248280837